### PR TITLE
Case study dailymotion

### PR DESCRIPTION
**Pull request by atarora**

### CODE REVIEW

File: qdrant-landing/content/blog/case-study-dailymotion.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+- Video content based recommendation system required processing all the video embedding at scale and in real time, as soon as they are added to the platform
+- Exact neighbour search at the scale and keeping them up to date with new video updates in real time at Dailymotion was unreasonable and unrealistic
+- Precomputed [KNN](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm) would be expensive and may not work due to video updates every hour
+- Platform needs fast recommendations ~ &lt; 100ms
+- Needed fast ANN search on a vector search engine which could support the scale and performance requirements of the platform
+
+### Background / Journey
+
+The quest of Dailymotion to deliver an intelligent video recommendation engine to deliver a curated selection of videos to its users started with a need to present more relevant videos to the first-time users of the platform (cold start problem) and implement an ideal home feed experience to allow users to watch videos that are expected to be relevant, diverse, explainable, and easily tunable. \
```
Comment (kacperlukawski) [inline qdrant-landing/content/blog/case-study-dailymotion.md]: Could we change one of the occurrences of "deliver" to something different? 

```suggestion
The quest of Dailymotion to deliver an intelligent video recommendation engine providing a curated selection of videos to its users started with a need to present more relevant videos to the first-time users of the platform (cold start problem) and implement an ideal home feed experience to allow users to watch videos that are expected to be relevant, diverse, explainable, and easily tunable. \
```

File: qdrant-landing/static/case-studies/dailymotion/solution-at-glance.png
Comment (kacperlukawski) [inline qdrant-landing/static/case-studies/dailymotion/solution-at-glance.png:1]: Shouldn't the user be described as "User A"? "Recommend to User A" suggests that

File: qdrant-landing/content/blog/case-study-dailymotion.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+   [Qdrant’s benchmarks](https://qdrant.tech/benchmarks/) has definitely been one of the key motivations for the Dailymotion’s team to try the solution and the team comments that the performance has been only better than the benchmarks. 
+
+
+**7. Ease of usage:**
+
+   Qdrant API’s have been immensely easy to get started with as compared to Google Vertex Matching Engine (which was Dailymotion’s initial choice) and the support from the team has been of a huge value to us.
+
+
+**8. Being able to fetch data by id:** 
+   Qdrant allows to retrieve vector point / videos by ids while the Vertex Matching Engine requires a vector input to be able to search for other vectors which was another really important feature for Dailymotion
```
Comment (kacperlukawski) [inline qdrant-landing/content/blog/case-study-dailymotion.md]: The formatting is different comparing to the other entries.

```suggestion
**8. Being able to fetch data by id** 

   Qdrant allows to retrieve vector point / videos by ids while the Vertex Matching Engine requires a vector input to be able to search for other vectors which was another really important feature for Dailymotion
```

File: qdrant-landing/content/blog/case-study-dailymotion.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+3. The overall huge performance gain on the low signal videos has contributed to more than 3 times increase on the interaction and CTR ( number of clicks) on the recommended videos.
+4. Seamlessly solved the initial cold start and low performance problems with the fresh content.
+
+### Outlook / Future plans
+
+The team is very excited with the results they achieved on their recommender system and wishes to continue building with it. \
+They aim to work on Perspective feed next and say
+
+>We've recently integrated this new recommendation system into our mobile app through a feature called Perspective. The aim of this feature is to disrupt the vertical feed algorithm, allowing users to discover new videos. When browsing their feed, users may encounter a video discussing a particular movie. With Perspective, they have the option to explore different viewpoints on the same topic. Qdrant plays a crucial role in this feature by generating candidate videos related to the subject, ensuring users are exposed to diverse perspectives and preventing them from being confined to an echo chamber where they only encounter similar viewpoints. \
```
Comment (kacperlukawski) [inline qdrant-landing/content/blog/case-study-dailymotion.md]: Previous quote had a different formatting.

```suggestion
>”We've recently integrated this new recommendation system into our mobile app through a feature called Perspective. The aim of this feature is to disrupt the vertical feed algorithm, allowing users to discover new videos. When browsing their feed, users may encounter a video discussing a particular movie. With Perspective, they have the option to explore different viewpoints on the same topic. Qdrant plays a crucial role in this feature by generating candidate videos related to the subject, ensuring users are exposed to diverse perspectives and preventing them from being confined to an echo chamber where they only encounter similar viewpoints.” \
```

**Review by kacperlukawski — COMMENTED** [review]
Really great work! I made a few comments, mostly about the formatting. Can we also get rid of the duplicated images? The `blog/case-study-dailymotion` directory could be removed, as it seems none of the images is displayed from that location.

File: qdrant-landing/static/case-studies/dailymotion/Dailymotion-Case-Study.jpg
[the remark this answers is not in the document]
Comment (atarora) [reply qdrant-landing/static/case-studies/dailymotion/Dailymotion-Case-Study.jpg:1]: This was provided by Manuel, I can request.

**Review by kacperlukawski — APPROVED** [review]

File: qdrant-landing/content/blog/case-study-dailymotion.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+- **Required response time < 100 ms**
+
+
+### Challenge
+
+
+- **Improve video recommendations** across all 3 applications of Dailymotion (mobile app, website and embedded video player on all major French and International sites) as it is the main driver of audience engagement and revenue stream of the platform.
+- Traditional [collaborative recommendation model](https://en.wikipedia.org/wiki/Collaborative_filtering) tends to recommend only popular videos, fresh and niche videos suffer due to zero or minimal interaction
+- Video content based recommendation system required processing all the video embedding at scale and in real time, as soon as they are added to the platform
+- Exact neighbour search at the scale and keeping them up to date with new video updates in real time at Dailymotion was unreasonable and unrealistic
```
Comment (mjang) [inline qdrant-landing/content/blog/case-study-dailymotion.md:46]: correcting spelling error (we use US English)
```suggestion
- Exact neighbor search at the scale and keeping them up to date with new video updates in real time at Dailymotion was unreasonable and unrealistic
```

File: qdrant-landing/content/blog/case-study-dailymotion.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+>”We've recently integrated this new recommendation system into our mobile app through a feature called Perspective. The aim of this feature is to disrupt the vertical feed algorithm, allowing users to discover new videos. When browsing their feed, users may encounter a video discussing a particular movie. With Perspective, they have the option to explore different viewpoints on the same topic. Qdrant plays a crucial role in this feature by generating candidate videos related to the subject, ensuring users are exposed to diverse perspectives and preventing them from being confined to an echo chamber where they only encounter similar viewpoints.” \
+> Gladys Roch - Machine Learning Engineer 
+
+
+
+![perspective-feed-with-qdrant](/case-studies/dailymotion/perspective-feed-qdrant.jpg)
+
+
+The team is also interested in leveraging advanced features like [Qdrant’s Discovery Api](https://qdrant.tech/documentation/concepts/explore/#recommendation-api) to promote exploration of content to enable finding not only similar but dissimilar content too by using positive and negative vectors in the queries and making it work with the existing collaborative recommendation model.
```
Comment (mjang) [inline qdrant-landing/content/blog/case-study-dailymotion.md:170]:
```suggestion
The team is also interested in leveraging advanced features like [Qdrant’s Discovery API](https://qdrant.tech/documentation/concepts/explore/#recommendation-api) to promote exploration of content to enable finding not only similar but dissimilar content too by using positive and negative vectors in the queries and making it work with the existing collaborative recommendation model.
```

**Pull request merged**